### DISCUSSION TITLE
archery支持多节点部署吗？

### DISCUSSION THREAD

**Discussion by peixubin** [hhyo/Archery#2598] [Q&A] [answered]
为应对大用户量访问，需要部署多个节点，来提升整体性能。
参考archery架构，感觉q-cluster是个问题。

**Comment by LeoQuote** [comment]
需要将 qcluster , worker, django web, 静态文件分开部署, 可以考虑改一下 dockerfile 的 entrypoint, 改成 web, frontend, worker 三个 入口, 然后起服务的时候分别扩容这三个服务

**Comment by peixubin** [comment]
qcluster怎么拆分？是否要使用一个redis或mq之类的存储？

**Reply by LeoQuote** [reply, marked as answer]
qcluster 的配置里就已经写好了, 就是用一个 redis 的, 你只需要起多个实例就行
